Gemfile

```diff
@@ -41,7 +41,8 @@ gem 'aasm', '~> 4.12.0'
 gem 'rubyXL'
 gem 'ruby-progressbar'
 gem 'dotenv-rails'
-gem 'rexml'
+gem 'rexml', '>= 3.2.7'
+gem 'nokogiri', '>= 1.16.5'
```

Comment:
nokogiri is a dependency of another gem, so we typically don't add it.  instead just run "bundle update nokogiri" should update the Gemfile.lock without having to update the  Gemfile.

Reply:
Done, thanks!